--- a/os_dep/linux/ioctl_cfg80211.c
+++ b/os_dep/linux/ioctl_cfg80211.c
@@ -487,7 +487,9 @@ void rtw_cfg80211_ch_switch_notify(_adapter *adapter, u8 ch, u8 bw, u8 offset, u8 ht)
 	rtw_cfg80211_ch_switch_notify_wrapper(adapter);

 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0))
-	cfg80211_ch_switch_notify(adapter->pnetdev, &chdef, 0, 0);
+	cfg80211_ch_switch_notify(adapter->pnetdev, &chdef, 0);
 #else
 	cfg80211_ch_switch_notify(adapter->pnetdev, &chdef);
 #endif
